Eatery/Views/BRB/BRBHistoryTableViewCell.swift

```diff
@@ -13,6 +13,7 @@ class BRBHistoryTableViewCell: UITableViewCell {
     private var titleLabel: UILabel!
     private var subtitleLabel: UILabel!
     private var priceLabel: UILabel!
+    private var positive: Bool!
```

Comment:
We don't need this property; the label text color can be set inside the `configure` method.